File: Average-Value-of-Even-Numbers-That-Are-Divisible-by-Three.java
class Solution {
    public int averageValue(int[] nums) {
        int total = 0;
        int count = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] % 2 == 0 && nums[i] % 3 == 0) {
                total += nums[i];
                count++;
            }
        }
        return count == 0 ? 0 : total / count;
    }
}

File: Average-Value-of-Even-Numbers-That-Are-Divisible-by-Three.py
class Solution(object):
    def averageValue(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        total = 0
        count = 0
        for i in range(len(nums)):
            if nums[i] % 2 == 0 and nums[i] % 3 == 0:
                total += nums[i]
                count += 1
        if count == 0:
            return 0
        return total // count
